### ISSUE TITLE
failed to connect to aria2

### ISSUE THREAD

**Issue by TheGoldenToilet** [legend2ks/YoutubeDownloader#14]
I'll be honest I've never really build stuff before and I'm probably doing something wrong. I saw that ``\YoutubeDownloader-master\YoutubeApp\Downloader\Aria2.cs`` looks to the executable in ``./utils/aria2c.exe`` so I added a utils folder to ``\YoutubeDownloader-master\YoutubeApp\utils`` containing aria2c.exe. After that didn't work I ran aria manually with the command ``aria2c.exe -k 1M -j 1 --allow-overwrite=true --disable-ipv6 --allow-piece-length-change=true --enable-rpc --rpc-listen-port=6888`` but that still gives me the same error

**Comment by legend2ks** [comment]
The `utils` folder should be placed next to the final built file:
```
YoutubeApp/bin/Release/net8.0/
|-- YoutubeApp.exe
|-- utils/
|   |-- aria2c.exe
|   |-- ffmpeg.exe
|   |-- yt-dlp.exe
```

**Comment by TheGoldenToilet** [comment]
After having placed the aria2c executable into `YoutubeDownloader-master\YoutubeApp\bin\Release\net8.0\utils` along with ffmpeg.exe and yt-dlp.exe. 
The reprogram not longer showed an error message after starting however, attempting to download videos would result the video to show `1 Error(s)` in the grabber list. After running ffmpeg in the utils folder from cmd it complained about not being to find some files in `../lib`, so I added the lib folder from chocolatey (where I got yt-dlp and ffmpeg), after  that both ffmpeg and yt-dlp worked prompting the url when ran. 
This hasn't solved the issue either, when attempting to download the grabber list shows `Fetching... 1/1` followed by `Finished 1/1     1 Error(s).`
After deleting the log files from the debug folder, and running the release version as administrator it mysteriously works now. I am no longer able to recreate the issue. It might have been me not realising that clicking the x button leaves a background process running.
